atlassian_jwt_auth/contrib/flask_app/utils.py

```diff
@@ -0,0 +1,30 @@
+from flask import current_app
+import jwt.exceptions
+import requests.exceptions
+
+
+def parse_jwt(verifier, encoded_jwt):
+    """Decode an encoded JWT using stored config."""
+    try:
+        claims = verifier.verify_jwt(
+            encoded_jwt,
+            current_app.config.asap['VALID_AUDIENCE'],
+            leeway=60,
+            verify=current_app.config.asap.get('VERIFY_TLS_CERT', True)
```

Comment:
I'm going to push back on this one. I don't see any reason to make a developer spin up generate a TLS server and start a web server just to run a test. Its very common to make this available to the user for this purpose.

What about compromising by emitting a warning whenever `VERIFY_TLS_CERT` is `False` and `debug` is not `True`?